api/catalogue/graphql.go

```diff
@@ -19,18 +17,11 @@ import (
 var schema string
 
 type gql struct {
```

Comment:
Maybe change this to GQL, to fit under convention as well?

Reply:
I don't want to export it